src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerPrototype.cs

```diff
@@ -30,6 +35,9 @@ internal class ServerPrototype
 
         private ConfigProperty serverMinMemoryProperty;
         private ConfigProperty serverMaxMemoryProperty;
+        BitArray initialIOAffinityArray;
+        private const int MAX64CPU = 64;
+        private const int MAX32CPU = 32;
```

Comment:
Since these values are also used in ServerProcessor, could you consolidate them in a common place? Like as static constants in the ServerProcessor class